terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -298,6 +321,146 @@ resource "kubernetes_cron_job" "workflow_manager" {
+    namespace = var.kubernetes_namespace
+  }
+
+  spec {
+    replicas = var.intake_batch_replica_count
+    selector {
+      match_labels = {
+        name = "intake-batch-${var.ingestor}"
+      }
+    }
+    template {
+      metadata {
+        labels = {
+          name = "intake-batch-${var.ingestor}"
+        }
+      }
+      spec {
+        service_account_name = module.account_mapping.kubernetes_account_name
+        container {
+          name  = "facile-container"
+          image = "${var.container_registry}/${var.facilitator_image}:${var.facilitator_version}"
+          args  = ["intake-batch-worker"]
+          resources {
+            # Batch intake is single-threaded, so we request an entire CPU for
+            # maximum throughput. We never expect to see batches larger than
+            # 3-400 MB, and batches are processed in memory, so request enough
+            # for whole ingestion batches plus a safety margin.
+            requests {
+              memory = "500Mi"
+              cpu    = "1.0"
+            }
+            limits {
+              memory = "550Mi"
+              cpu    = "1.5"
+            }
+          }
+          env {
+            name = "BATCH_SIGNING_PRIVATE_KEY"
+            value_from {
+              secret_key_ref {
+                name     = kubernetes_secret.batch_signing_key.metadata[0].name
+                key      = "secret_key"
+                optional = false
+              }
+            }
+          }
+          env {
+            name = "PACKET_DECRYPTION_KEYS"
+            value_from {
+              secret_key_ref {
+                name     = var.packet_decryption_key_kubernetes_secret
+                key      = "secret_key"
+                optional = false
+              }
+            }
+          }
+          env_from {
+            config_map_ref {
+              name     = kubernetes_config_map.intake_batch_config_map.metadata[0].name
+              optional = false
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+resource "kubernetes_deployment" "aggregate" {
+  metadata {
+    name      = "aggregate-${var.ingestor}"
+    namespace = var.kubernetes_namespace
+  }
+
+  spec {
+    replicas = var.aggregate_replica_count
+    selector {
+      match_labels = {
+        name = "aggregate-${var.ingestor}"
+      }
+    }
+    template {
+      metadata {
+        labels = {
+          name = "aggregate-${var.ingestor}"
+        }
+      }
+      spec {
+        service_account_name = module.account_mapping.kubernetes_account_name
+        container {
+          name  = "facile-container"
+          image = "${var.container_registry}/${var.facilitator_image}:${var.facilitator_version}"
+          args  = ["aggregate-worker"]
+          resources {
+            # As in the intake-batch case, aggregate jobs are single threaded
+            # and need to fit whole ingestion batches into memory.
+            requests {
```

Comment:
Same as above, I'd go to the minimal requirements for small batches on the request side to start with.